src/main/resources/export-build.qute.gradle

```diff
@@ -34,4 +35,5 @@ dependencies {
 
 application {
 	mainClass = '{fullClassName}'
+	applicationDefaultJvmArgs = []
```

Comment:
this really use get values for runtime options rather than just setting it blank.

Reply:
Moving RUNTIME_OPTIONS -> applicationDefaultJvmArgs to a separate pull request. The current one refocusses on mavenLocal() support only.